DBUtils/api-src/org/labkey/dbutils/api/SimpleQueryUpdater.java

```diff
@@ -191,14 +196,13 @@ else if (insertedRows.size() != rows.size()) {
         return updatedRows;
     }
 
-    public List<CaseInsensitiveMapWrapper<Object>> delete(Map<String, Object>... rowArray) throws QueryUpdateServiceException, SQLException, InvalidKeyException, BatchValidationException {
-        List<Map<String, Object>> rowsToDelete = makeRowsCaseInsensitive(rowArray);
+    public List<CaseInsensitiveMapWrapper<Object>> delete(List<Map<String, Object>> rows) throws QueryUpdateServiceException, SQLException, InvalidKeyException, BatchValidationException {
```

Comment:
would this change any other places where we use SimpleQueryUpdater?

Reply:
No, this doesn't change any existing functionality. It only adds the ability to pass a list of rows for deletion. All existing code will continue to function in the exact same way as previously